packages/libthread-db/build.sh

```diff
@@ -0,0 +1,21 @@
+TERMUX_PKG_HOMEPAGE=https://android.googlesource.com/platform/ndk/
+TERMUX_PKG_DESCRIPTION="Thread debugging library"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=22
```

Comment:
Also mention that libthread-db was removed in NDK r23

Reply:
Fixed.